test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -217,8 +216,9 @@ class OverrideHostTest : public XdsEnd2endTest {
            num_requests;
   }
 
-  static Route BuildStatefulSessionRouteConfig(absl::string_view match_prefix,
-                                               absl::string_view cookie_name) {
+  static Route BuildStatefulSessionRouteConfig(
+      absl::string_view match_prefix, absl::string_view cookie_name,
+      absl::optional<google::protobuf::Duration> opt_duration = absl::nullopt) {
```

Comment:
Suggest passing this in as `grpc_core::Duration` instead of `google::protobuf::Duration` and then converting to protobuf form inside this function.  That will make it a bit more ergonomic for callers.